device/samples/cloud_to_device_message.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+require('dotenv').config();
+const { Message } = require('azure-iot-device');
+const { Client : HubClient } = require('azure-iothub');
+
+// NOTE: The sample is for the Azure IoT Hub Service Client, for communicating with the Device Client.
+// This sample should be used in tandem with the simple_sample_device.js sample to send a message to the running device.
+// In this example we instantiate an iot hub client via a connection string and send a message from the hub to the listening device
+
+const generateImportantMessage = messageId => new Message(
+    JSON.stringify({ 
+        messageId : messageId,
+        data : 'beep boop bopity bop',
```

Comment:
This isn't a good way of setting stuff.  Message id's are properties of the message itself and not the data of the message.